sonic-xcvrd/tests/test_xcvrd.py

```diff
@@ -215,3 +215,103 @@ def test_post_port_sfp_dom_info_to_db(self):
     def test_init_port_sfp_status_tbl(self):
         stop_event = threading.Event()
         init_port_sfp_status_tbl(stop_event)
+
+    @patch('xcvrd.xcvrd_utilities.y_cable_helper.y_cable_platform_sfputil', MagicMock(return_value=[0]))
+    @patch('y_cable_helper.logical_port_name_to_physical_port_list', MagicMock(return_value=[0]))
+    @patch('y_cable_helper._wrapper_get_presence', MagicMock(return_value=True))
+    @patch('y_cable_helper.get_muxcable_info', MagicMock(return_value={'tor_active': 'True',
+                                                                       'mux_direction': '1',
```

Comment:
Shouldn't `mux_direction` be a string like `self`?

Reply:
fixed